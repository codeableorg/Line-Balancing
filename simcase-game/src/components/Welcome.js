/** @jsx jsx */

import { jsx } from "@emotion/core";

function Welcome() {
  const contentStyle = {
    padding: 10
  };
  return (
    <main css={contentStyle}>
      <h2>Welcome</h2>
      <img
        src="assets/img/wharton_logo.png"
        alt="Welcome"
        css={{ width: 300 }}
      />
      <p id="welcome">Welcome message for students</p>
    </main>
  );
}

export default Welcome;
