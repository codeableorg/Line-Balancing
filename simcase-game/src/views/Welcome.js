/** @jsx jsx */
import React from "react";
import { jsx } from "@emotion/core";
import { Link } from "@reach/router";

import Navbar from "../components/Navbar";
import { Button } from "../components/ui";
import { MainContent } from "../components/helpers";
import fire from "../firebase";

const container = {
  display: "flex",
  flexDirection: "column",
  justifyContent: "space-evenly",
  alignItems: "center",
  textAlign: "center",
  color: "rgba(0, 0, 0, 0.541327)",
  p: {
    fontSize: "16px",
    lineHeight: "24px",
    width: "257px",
    height: "110px"
  },
  img: {
    width: "100px"
  }
};

function Welcome() {
  return (
    <>
      <Navbar showDialogValue={true} modalValue="role" />
      <MainContent styles={container}>
        <img src="/stopwatch.png" alt="Welcome" />
        <p>
          Balance production to increase production. Use the radio buttons to
          shift tasks between adjacent workers. Good Luck!
        </p>
        <Button>
          <Link to="/game/1">START</Link>
        </Button>
      </MainContent>
    </>
  );
}

export default Welcome;
