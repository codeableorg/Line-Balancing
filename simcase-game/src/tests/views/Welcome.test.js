import React from "react";
import { render } from "@testing-library/react";

import Welcome from "../../views/Welcome";

test("rendering Welcome", () => {
  const { asFragment } = render(<Welcome />);
  expect(asFragment()).toMatchSnapshot();
});
