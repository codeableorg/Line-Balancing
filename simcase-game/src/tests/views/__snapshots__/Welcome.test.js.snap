// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering Welcome 1`] = `
<DocumentFragment>
  <header
    class="css-1ienwms-Header"
  >
    <div
      class="css-1755hce-NavBar"
    >
      <h2>
         
      </h2>
    </div>
    <div
      class="css-1755hce-NavBar"
    >
      <a
        class="css-109fdo4-NavBar"
        href="/"
      >
        <img
          alt="Wharton Logo"
          class="css-uj5uly-NavBar"
          src="/assets/img/wharton_logo.png"
        />
        <h1
          class="css-1lp1hih-NavBar"
        >
          Line Balancing
        </h1>
      </a>
    </div>
    <div
      class="css-191exao-NavBar"
    >
      <button
        class="css-1ww68zw-NavBar"
      >
        ☰
      </button>
    </div>
  </header>
  <main
    class="css-1vaqpk9-MainContent"
  >
    <h2>
      Welcome
    </h2>
    <img
      alt="Welcome"
      src="assets/img/wharton_logo.png"
    />
    <p>
      Welcome message for students
    </p>
  </main>
  <section
    class="css-3vj8vd-Action"
  >
    <button
      class="css-1rlcy0k-Button"
    >
      <a
        href="/game/1"
      >
        Start Game
      </a>
    </button>
  </section>
</DocumentFragment>
`;
